Tests/StoreKitUnitTests/PurchasesOrchestratorSK1Tests.swift

```diff
@@ -375,14 +480,15 @@ class PurchasesOrchestratorSK1Tests: PurchasesOrchestratorTestCase {
                                                    product: StoreProduct(sk1Product: product),
                                                    completion: completion)
             }
+            fail("Expected error")
```

Comment:
Added these exceptions in a bunch of places as otherwise the expectation below would not be executed if the method didnt throw